src/db.py

```diff
@@ -741,6 +812,10 @@ def _upgrade(oldversion):
                 # player id as a string). When nocasing players, this may cause some records to be merged.
                 with open(os.path.join(dn, "db", "upgrade2.sql"), "rt") as f:
                     c.executescript(f.read())
+            if oldversion < 3:
+                print ("Upgrade from version 2 to 3...", file=sys.stderr)
+                with open(os.path.join(dn, "db", "upgrade3.sql"), "rt") as f:
+                    c.executescript(f.read())
```

Comment:
Hmm, this possibility doesn't appear to me right now, but you're the DB pro. There are only two upgrade paths right now anyway, so it's not so bad. When we get to 4-5 that are all the same thing I might bring this up again :)